transaction/core/src/ring_signature/rct_bulletproofs.rs

```diff
@@ -295,10 +437,7 @@ fn sign_with_balance_check<CSPRNG: RngCore + CryptoRng>(
     //
     // Note: This implicit fee output is not the same as the accumulated fee output
     // produced by the enclave -- the blinding of that output is not zero.
-    let sum_of_output_blindings: Scalar = output_values_and_blindings
-        .iter()
-        .map(|(_, blinding)| blinding)
-        .sum();
+    let sum_of_output_blindings: Scalar = output_secrets.iter().map(|secret| secret.blinding).sum();
```

Comment:
Do we need to worry about overflowing here?

Reply:
Overflowing definitely happens, and it's supposed to wrap around mod p -- the dalek Scalar takes care of that